### ISSUE TITLE
Appveyor job throwing error on hugo command

### ISSUE THREAD

**Issue by mattstratton** [devopsdays/devopsdays-web#1664]
```
Build started
git clone -q --branch=dodAMS-2017-content-chgs https://github.com/devopsdays/devopsdays-web.git C:\projects\devopsdays-web
git checkout -qf 2081c53942d173e6198a0c8798697b190c31dd08
Running Install scripts
choco install hugo -y
Chocolatey v0.10.3
Installing the following packages:
hugo
By installing you accept licenses for the packages.
hugo v0.18.1 [Approved]
hugo package files install completed. Performing other installation steps.
Downloading hugo 64 bit
  from 'https://github.com/spf13/hugo/releases/download/v0.18.1/hugo_0.18.1_Windows-64bit.zip'
Progress: 100% - Completed download of C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\hugo\0.18.1\hugo_0.18.1_Windows-64bit.zip (5.11 MB).
Download of hugo_0.18.1_Windows-64bit.zip (5.11 MB) completed.
Hashes match.
Extracting C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\hugo\0.18.1\hugo_0.18.1_Windows-64bit.zip to C:\ProgramData\chocolatey\lib\hugo\tools...
C:\ProgramData\chocolatey\lib\hugo\tools
 ShimGen has successfully created a shim for hugo_0.18.1_windows_amd64.exe
 The install of hugo was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\hugo\tools'
Chocolatey installed 1/1 packages. 0 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
hugo version
'hugo' is not recognized as an internal or external command,
operable program or batch file.
Command exited with code 1
```

**Comment by mattstratton** [comment]
Until the chocolatey package is updated to rename the hugo executable, we will need to refer to it explicitly, which isn't good for versions.

**Comment by mattstratton** [comment]
Fixed in 3a819d0bcf5172ab11c75b9008664bd0d6e0edce

I'm leaving this open until the [chocolatey package](https://chocolatey.org/packages/hugo) gets updated.

**Comment by chantisnake** [comment]
I am the maintainer of Hugo. I saw your message. I personally think the original renaming command is not secure (there will be error if there is more than one exe in the root.

I will find a way soon.

I also recommend you guys to find a way to adopt the long name.

**Comment by mattstratton** [comment]
@chantisnake thanks! I worked around it by explicitly referring to the executable, but my concern is that when a new version of hugo is released, that name will change and chocolatey will dump a different exe in there.

I suspect the safest thing is to call the specific version of the chocolatey package, something like this:

`choco install hugo -y --version 0.8.1`

Because then I'll know the exact exe name, etc. Seem logical?

**Comment by chantisnake** [comment]
I have checked the zip file.

Can you try using PowerShell to find the executable for now?

$hogoExe = get-command hugo*.exe

**Comment by chantisnake** [comment]
this is fixed.

**Comment by bridgetkromhout** [comment]
I see an error: https://ci.appveyor.com/project/DevOpsDays/devopsdays-web/build/1.0.62
```
hugo_0.18.1_windows_amd64.exe version
'hugo_0.18.1_windows_amd64.exe' is not recognized as an internal or external command,
operable program or batch file.
Command exited with code 1
```

**Comment by mattstratton** [comment]
Yep. I have to fix the appveyor job now that the chocolatey package is updated.

**Comment by mattstratton** [comment]
This was actually fixed in 2acd5d3fa5b4a1231a9a56074a54c9312c65f05e
